src/Simplex/Chat/Store/SQLite/Migrations/M20250526_short_links.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Store.SQLite.Migrations.M20250526_short_links where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20250526_short_links :: Query
+m20250526_short_links =
```

Comment:
we already had migration with that name I think... Maybe short_link_profiles

Reply:
it's of different date, so name is different